ansys/mapdl/core/mapdl.py

```diff
@@ -2225,19 +2315,38 @@ def load_table(self, name, array, var1="", var2="", var3=""):
         >>> mapdl.parameters['MY_TABLE']
         array([0.0001, 0.0001, 0.0005, 0.0005, 0.0002, 0.0002])
         """
-        if array.ndim < 2:
+        if not isinstance(array, np.ndarray):
+            raise ValueError("The table should be a Numpy array")
+        if array.shape[0] < 2 or array.shape[1] < 2:
+            raise ValueError("One or two of the array dimensions are too small to create a table.")
+        if array.ndim == 2:
+            self.dim(name, "TABLE", imax=array.shape[0]-1, jmax=array.shape[1]-1, kmax="",
+                     var1=var1, var2=var2, var3=var3, csysid=csysid)
+        elif array.ndim == 3:
+            self.dim(name, "TABLE", imax=array.shape[0]-1, jmax=array.shape[1]-1, kmax=array.shape[2],
+                     var1=var1, var2=var2, var3=var3, csysid=csysid)
```

Comment:
Get in the habit of providing whitespace around operators like `-`.
```suggestion
            self.dim(name, "TABLE", imax=array.shape[0]-1, jmax=array.shape[1] - 1, kmax="",
                     var1=var1, var2=var2, var3=var3, csysid=csysid)
        elif array.ndim == 3:
            self.dim(name, "TABLE", imax=array.shape[0] - 1, jmax=array.shape[1] - 1, kmax=array.shape[2],
                     var1=var1, var2=var2, var3=var3, csysid=csysid)
```

Reply:
Sure thing!